packages/helpers/packages/CookieHelpers/index.js

```diff
@@ -32,3 +32,13 @@ export const setCookie = (name, value, expireDays = 90, path = '/') => {
   const expires = `expires=${d.toUTCString()}`;
   document.cookie = `${name}=${value};${expires};path=${path}`;
 };
+
+/**
+ * delete a cookie
+ * @author Reza Erami
+ * @param name
+ */
+export const deleteCookie = (name) => {
+  const expires = `expires=${0}`;
+  document.cookie = `${name}=${null};${expires}`;
+};
```

Comment:
This changes are not a part of this PR, it is better to have PRs which do only do one job